site/_utils/add-pagination.js

```diff
@@ -38,49 +40,49 @@ const createHref = (baseUrl, i) => baseUrl + (i === 0 ? '/' : `/${i + 1}/`);
 const createPermalink = (baseUrl, i) => createHref(baseUrl, i) + 'index.html';
 
 /**
- * Take array of elements and returns an array of paginated pages for the elements.
- * @param {any[]} items Collection item's for pagination.
- * @param {string} baseUrl Beginning of url.
- * @param {{[key: string]: unknown}} additional
- * @return {PaginatedPage[]}
+ * Take array of elements of an item and returns an array of paginated pages for that item.
+ * @param {TODO} item Item to paginate.
```

Comment:
can you type this?

Reply:
I don't want to yet. Right now the function is used by authors and tags but their data sources are different. I want to get tags to use the same data structure first and then type these functions. You'll see in some places I kind of massage or ignore the typings in order to make it work for both data types without having to write some crazy checks so the linter doesn't complain.